src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form_field {
+  margin-bottom: 20px;
+}
+
+.form_field:not(:last-child) {
+  margin-top: 20px;
+}
+
+.search_bar {
+  position: relative;
+  width: 100%;
+}
+
+.image_big {
+  position: absolute;
+  display: inline-block;
+  width: 19px;
+  height: 19px;
+  left: 25px;
+  top: -1px;
+}
+
+.image_small {
+  position: absolute;
+  display: inline-block;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 4px;
+}

```

Comment:
Combine all repeatable styles for small and big inputs

Reply:
done